app/models/attributes/policies_config.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Attributes::PoliciesConfig < ActiveRecord::Type::Text
+  def cast(value)
+    if value.is_a?(Proxy::PoliciesConfig)
+      value
+    else
+      Proxy::PoliciesConfig.new(value)
+    end
+  end
+
+  def serialize(value)
+    value.to_json
+  end
+
+  def changed_in_place?(raw_old_value, new_value)
+    new_value != cast(raw_old_value)
+  end
```

Comment:
I think this is not needed if you have `deserialize` method

Reply:
Will try. Thanks.

Reply:
`#deserialize` by default delegates to `#cast`. So it is always defined. This, as far as source comment reads, is needed in case something changed underlying object in place. Not sure anything can do that. I added it just in case.

from active_model/type/value.rb:
```

      # Determines whether the mutable value has been modified since it was
      # read. Returns +false+ by default. If your type returns an object
      # which could be mutated, you should override this method. You will need
      # to either:
      #
      # - pass +new_value+ to Value#serialize and compare it to
      #   +raw_old_value+
      #
      # or
      #
      # - pass +raw_old_value+ to Value#deserialize and compare it to
      #   +new_value+
      #
      # +raw_old_value+ The original value, before being passed to
      # +deserialize+.
      #
      # +new_value+ The current value, after type casting.
      def changed_in_place?(raw_old_value, new_value)
        false
      end
```